Makefile-ponyc

```diff
@@ -51,6 +51,16 @@ tune ?= generic
 cpu ?= $(arch)
 fpu ?=
 bits ?= $(shell getconf LONG_BIT)
+staticbuild ?= false
```

Comment:
so we have a --static flag for ponyc.

i think for more symmetry, that calling this "static" might be better.